app/workers/mhv/audit_login_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module MHV
+  class AuditLoginJob
+    include Sidekiq::Worker
+
+    sidekiq_options retry: false
+
+    def perform(uuid)
+      user = User.find(uuid)
+
```

Comment:
Total nit, but it might be a good idea to also have the:

`return false if user.mhv_last_signed_in` check here as added precaution since we're making the call async. Similar comment below for Logout.